tools/version/src/main/java/org/apache/polaris/version/PolarisVersion.java

```diff
@@ -145,9 +145,11 @@ public static String readSourceLicenseFile() {
     return readResource("LICENSE");
   }
 
+  /*
```

Comment:
Why would this be part of the version? This cannot be accurate because this class doesn't know what was bundled or how dependencies were pulled in.